src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/StaticHttpSettings.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Http
+{
+    internal static class StaticHttpSettings
+    {
+        private const string AllowNonAsciiCharactersEnvironmentVariableSettingName = "DOTNET_SYSTEM_NET_HTTP_SOCKETSHTTPHANDLER_ALLOWNONASCIIHEADERS";
```

Comment:
Would these all be more accurate as `AllowLatin1Headers`?

Reply:
In theory it could be any single-byte `ISO-8859-*`  encoding, not just Latin1, but probably still makes more sense, than "non-ascii".

Reply:
Renamed.